### ISSUE TITLE
rollbar.js reporting error incorrectly as unknown script error on Chrome, but not Firefox

### ISSUE THREAD

**Issue by and-megan** [rollbar/rollbar.js#505]
tl;dr; HTML in iframe. Firefox points to the correct line, Chrome reports as: File "(unknown)" line (unknown) col 0 in [anonymous]. rollbar.js v2.2.10.

Yep so again this was the basic html I am running:

<html>

<body>

<iframe sandbox="allow-scripts allow-popups allow-same-origin allow-popups-to-escape-sandbox" src="http://localhost:8081/internal/testing/throw-error/index.html">

</iframe>

</body>

</html>

And inside a simple local static node server the javascript looks like (rollbar.js is very similar to the boilerplate included in the javascript integration guide):

require('../../../javascript/rollbar.js');

throw new Error("please show line info");

So I've attached how it looks when paused in Chrome 62 on the exception and then stepping into minified rollbar.

As you can see the exception object in chrome has no fields and it hasn't been passed to rollbar correctly in the window.onerror handler.

Whereas in the last screenshot you can see that Firefox was happy to include line number and other information.

x[0] = " ";

https://downloads.intercomcdn.com/i/o/39757210/0dda9d4b01fd37ca680723cb/Screen+Shot+2017-11-17+at+10.15.48.png
Chrome showing unknown script error: https://downloads.intercomcdn.com/i/o/39757211/79529ff6aa59ed2271438e72/Screen+Shot+2017-11-17+at+10.18.08.png
Firefox showing correct line number: https://downloads.intercomcdn.com/i/o/39757213/0aff211004dd98d0b3ea2ae8/Screen+Shot+2017-11-17+at+10.16.33.png

**Comment by rokob** [comment]
Sounds like a Chrome issue?

**Comment by rivkahstandig3636** [comment]
Closing as this is an issue with Chrome and not rollbar.js

**Comment by dasDaniel** [comment]
so is chrome not supported?

**Comment by rivkahstandig3636** [comment]
Chrome is supported, but we cannot fix their bugs for them.
